bindings/matrix-sdk-ffi/src/api.udl

```diff
@@ -1,9 +1,19 @@
 namespace sdk {
+    void setup_tracing(LogLevel log_level);
```

Comment:
Hmm, why not a full tracing configuration string that can also be `matrix_sdk=debug,info` or sth. like that?

Reply:
Then the configuration for the no-longer-used crate would be a no-op and the replacement crate would use the default log level. It would still be much more flexible.

Reply:
Alrighty, let's do that then 👍